ark/utils/metacluster_remap_gui/test_colormap_helper.py

```diff
@@ -0,0 +1,9 @@
+from .colormap_helper import distinct_cmap, distinct_rgbs
+
+
+def test_colormap_is_distinct():
```

Comment:
for test functions, please name them original_function_name_test

Reply:
After testing this change, pytest does not collect function unless they have a `test_` prefix.

Additionally I checked and it seems the other tests start with `test_` 
https://github.com/angelolab/ark-analysis/search?p=2&q=test
![image](https://user-images.githubusercontent.com/6101677/134821732-f7838baa-50c7-4edb-ae28-cde5cbebf2f9.png)

Reply:
Yup, totally right. It should be test_function_name, not function_name_test